### DISCUSSION TITLE
New Release page is not responsive

### DISCUSSION THREAD

**Discussion by landsman** [community/community#109682] [Repositories]
Here is a problem with one div which breaks the page, check it:

![image](https://github.com/community/community/assets/3184228/6a1e1c69-5808-4ebc-bde5-10e6215ee302)

![image](https://github.com/community/community/assets/3184228/5a589951-ed96-4373-acb0-c9fe4f4c23d8)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.150.0 (123387055)
Device: iPhone 13 mini (iOS 17.2.1)
</details>

**Comment by nightcomet919** [comment]
Please explain in more detail

**Comment by iskae** [comment]
Hi @landsman 👋 

Thank you for reaching out, but unfortunately the `Mobile` category is reserved for our native mobile apps on iOS and Android. It looks like you are using our Web App in mobile device, therefore we won't be able to help you further on this 🙇 

You can reach out to our colleagues in [Repositories](https://github.com/orgs/community/discussions/categories/repositories) for this request 🙇

**Comment by landsman** [comment]
category changed
